mteb/cli/build_cli.py

```diff
@@ -210,19 +202,23 @@ def _add_run_parser(subparsers: argparse._SubParsersAction) -> None:
         "--batch_size",
         type=int,
         default=None,
-        help="Batch size of the encode. Will be passed to the MTEB as MTEB.evaluate(model, encode_kwargs = {'batch_size': value}).",
+        help="Batch size of the encode. Will be passed to the MTEB as mteb.evaluate(model, task, encode_kwargs = {'batch_size': value}).",
     )
     parser.add_argument(
-        "--overwrite",
-        action="store_true",
-        default=False,
-        help="Overwrite the output file if it already exists",
+        "--overwrite_strategy",
+        type=str,
+        default="only-missing",
+        choices=["always", "never", "only-missing"],
```

Comment:
I think we can create list directly from `OverwriteStrategy`
```suggestion
        choices=[val.value for val in OverwriteStrategy],
```